### ISSUE TITLE
Please remove grey overlay when dropdown-menu appears

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#121]
But... click on the outside the dropdown menu region should close the dropdown menu.
Keyboard accessibility should be guaranteed (if possible please focus the first element)
ESC should close the dropdown menu as well, and stay in the selected issue before.

Issue reference:
#60
